app/controllers/nonsipp/TaskListController.scala

```diff
@@ -302,16 +296,10 @@ object TaskListController {
       s"$prefix.title",
       TaskListItemViewModel(
         LinkMessage(
-          messageKey(prefix, "employercontributions.title", employerContributionStatus),
-          employerContributionStatus match {
-            // change so Completed goes to member list page
-            case _ =>
-              controllers.nonsipp.employercontributions.routes.EmployerContributionsController
-                .onPageLoad(srn, NormalMode)
-                .url
-          }
+          messageKey(prefix, "employercontributions.title", employerContributionStatusAndLink._1),
+          employerContributionStatusAndLink._2
         ),
-        employerContributionStatus
+        employerContributionStatusAndLink._1
```

Comment:
rather than calling the arity, could you either use a case class or destructure the `val employerContributionStatusAndLink` to `val (employerContributionStatus, val employerContributionLink) = ?`

Reply:
Changed.